app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -1,6 +1,7 @@
 module VerifySpAttributesConcern
   def needs_completion_screen_reason
     return nil if sp_session[:issuer].blank?
+    return nil if sp_session[:request_url].blank?
```

Comment:
What is the scenarios where there is an issuer but not a request URL?

Reply:
I have been digging into it quite a bit, and I'm still not sure.

Reply:
https://github.com/18F/identity-idp/pull/4848 made some changes in this area as well